Imperatus.rte/Devices/Weapons/Mauler/Mauler.lua

```diff
@@ -0,0 +1,31 @@
+function Create(self)
+	self.chainCount = 15;
+	self.spread = math.rad(self.ParticleSpreadRange);
+end
+function Update(self)
+	if self.FiredFrame then
+		local lastChainID = 0;
+		for i = 1, self.chainCount do
+			local chain = CreateMOSParticle("Particle Mauler Chainshot ".. i % 2 + 1);
+			chain.Pos = self.MuzzlePos;
+			chain.Vel = self.Vel + Vector(math.random(50, 70) * self.FlipFactor, 0):RadRotate(self.RotAngle + self.spread * 0.5 - self.spread * i/self.chainCount);
+			chain.PinStrength = 5000;	--Prevent any movement in the first frame
```

Comment:
Comment formatting consistency - pretty sure you normally have a space before the comment rather than a tab.

Reply:
Damn - I must've missed the tab vs. space thing, I hope it doesn't appear too often in scripts.